net-im/telegram-desktop/telegram-desktop-5.12.3-r6.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{11..13} )
+PYTHON_COMPAT=(python3_{11..13})
```

Comment:
I'll drop this change as usual Gentoo style is to keep the spaces and it's not related to the Qt issue.